### PR TITLE
Misc QOL enhancements

### PR DESCRIPTION
**Pull request by BrychanOdlum**

### CODE REVIEW

File: src/common.scss [hunk 1]
```diff
@@ -54,30 +52,13 @@ button {
...
   @import './styles/Vendor/react-contextmenu';
   @import './styles/Vendor/react-contextmenu';
   @import './styles/Vendor/github-fork-ribbon-css';
-}
-
-::-webkit-scrollbar {
-  width: 0px;
-  background: transparent; /* make scrollbar transparent */
-}
+}
```
Comment (raphaelvigee) [inline src/common.scss]: EOF

File: src/components/Core/Overview/OverviewPage.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+    this.setState({
+      frequentlyPlayed,
+      recentlyPlayed
+    })
+  }
+
+  render() {
+    const {frequentlyPlayed, recentlyPlayed} = this.state;
+    console.log(recentlyPlayed);
```
Comment (raphaelvigee) [inline src/components/Core/Overview/OverviewPage.jsx]: Remove `console.log`

File: src/components/Core/common/SongList/SongList.scss [hunk 3]
```diff
@@ -130,50 +133,53 @@
...
+      height: 8px;
+    }
+    50% {
+      height: 10px;
+    }
+    100% {
+      height: 5px;
+    }
   }
 }
```
Comment (raphaelvigee) [inline src/components/Core/common/SongList/SongList.scss]: EOF

File: src/components/common/Loader.scss [hunk 4]
```diff
@@ -1,41 +1,53 @@
...
-      top: -1px;
-      left: -1px;
-      width: 58px;
-      height: 58px;
-      opacity: 0;
+      height: 20px;
     }
   }
-}
+}
```
Comment (raphaelvigee) [inline src/components/common/Loader.scss]: EOF

File: src/common.scss (hunk 1 above)
Comment (BrychanOdlum) [reply src/common.scss]: has already been done

File: src/components/common/Loader.scss (hunk 4 above)
Comment (BrychanOdlum) [reply src/components/common/Loader.scss]: already done.

File: src/components/Core/common/SongList/SongList.scss (hunk 3 above)
Comment (BrychanOdlum) [reply src/components/Core/common/SongList/SongList.scss]: on it, will commit in 30 seconds.

File: src/components/Core/Overview/OverviewPage.jsx (hunk 2 above)
Comment (BrychanOdlum) [reply src/components/Core/Overview/OverviewPage.jsx]: shall do!

**Comment by BrychanOdlum** [comment]
@raphaelvigee all resolved.

**Pull request merged**